storybook/cypress/e2e/progress-indicator.cy.ts

```diff
@@ -0,0 +1,60 @@
+describe("Progress Indicator", () => {
+  const sizes = [
+    {
+      size: "small",
+      iconSize: 24,
+    },
+    {
+      size: "medium",
+      iconSize: 32,
+    },
+    {
+      size: "large",
+      iconSize: 48,
+    },
+  ];
+
+  for (const { size, iconSize } of sizes) {
+    it("should show small spinner", () => {
+      cy.visit(`http://localhost:45000/iframe.html?id=core-progress-indicator--${size}`)
+        .get("dso-progress-indicator")
+        .should("have.attr", "size", size)
+        .shadow()
+        .as("dsoProgressIndicator")
+        .then(($shadowRoot) => {
+          $shadowRoot.prepend("<style>.path { stroke-dasharray: 0 !important; }</style>");
```

Comment:
@LukeVerlaan Hier heb ik de style gewoon in het Shadow Root geknalt. Het nadeel van de media queries in de .scss files onderbrengen is dat ze ook in de buildartifacts bij de applicaties komen